sdk/iot/Azure.Iot.Hub.Service/src/Generated/ConfigurationRestClient.cs

```diff
@@ -487,8 +487,8 @@ internal HttpMessage CreateApplyOnEdgeDeviceRequest(string id, ConfigurationCont
             return message;
         }
 
-        /// <summary> Apply the provided configuration content to the specified edge device. </summary>
-        /// <param name="id"> The unique identifier of the device. </param>
+        /// <summary> Applies the provided configuration content to the specified edge device. Configuration content must have modules content For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version &apos;2020-03-13&apos;.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version &apos;2019-10-01&apos;. </summary>
+        /// <param name="id"> Device ID. </param>
```

Comment:
If possible, preferred casing is Id

Reply:
This is generated code from the autorest descriptions that we are already working on changing through the swagger itself. I don't see a need to override these descriptions with custom generator code unless the service folks reject our comment refactors